### PR TITLE
Add manual trigger for building the perfetto UI without releasing it

### PR DESCRIPTION
**Pull request by martin-fleck-at**

### CODE REVIEW

File: .github/workflows/build-perfetto-ui.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        id: sha
+        run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+      - name: Install Dependencies
+        run: tools/install-build-deps --ui
+      - name: Run build
+        run: ui/build
+      - name: Create archive
+        run: zip -r perfetto-ui-${{ steps.sha.outputs.sha_short }}.zip ui/wasm ui/css ui/out/tsc -x "ui/out/tsc/node_modules/*"
+      - name: Upload Perfetto UI
+        if: runner.os == 'Linux'
```
Comment (jfaltermeier) [inline .github/workflows/build-perfetto-ui.yml:24]: I don't think we need this if, because we have no build matrix with other OSes.

**Review by jfaltermeier — COMMENTED** [review]
Looks good to me, I only have a minor comment. 

I think we haven't testet the workflow yet? It might be that the upload artifact action will create a further archive of `perfetto-ui-${{ steps.sha.outputs.sha_short }}.zip`, so that we have a zip in a zip. But I am not 100% sure about this.

File: .github/workflows/build-perfetto-ui.yml (hunk 1 above)
Comment (martin-fleck-at) [reply .github/workflows/build-perfetto-ui.yml]: Good point, I removed it!

**Comment by martin-fleck-at** [comment]
@jfaltermeier Good point! I just tested it [here](https://github.com/eclipsesource/perfetto/actions/runs/5069325228) and it does indeed create a zip within a zip, I'll fix it.

**Comment by jfaltermeier** [comment]
I think it looks good now. I diffed the contents of the Zip with the last release and the only difference was the additional README. So when we remove the additional PR triggers again we may merge this imo.

**Review by jfaltermeier — APPROVED** [review]

**Pull request merged**